docs/content/v1/deep.md

```diff
@@ -1,27 +0,0 @@
----
-title: Getting Deep
-weight: 8
----
-
-## Native Web Components
-
-Because LEGO is actual native Web Components, all its native possibilities (like [slots](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_templates_and_slots#Adding_flexibility_with_slots)),
-[:host](https://developer.mozilla.org/en-US/docs/Web/CSS/:host)
-and whatever exists or will exist are on board.
-
-## Browser Compatibility
-
-LEGO is based on native _customElements_.
-[Support for customElement is spreading](https://caniuse.com/#feat=custom-elementsv1) and

```

Comment:
I have added the support to the introduction in the **README.md** and **welcome.md**.